roles/nova-common/defaults/main.yml

```diff
@@ -40,6 +40,7 @@ nova:
   install_packages:
     - python-numpy
     - virt-top
+    - python-ironicclient
```

Comment:
We install the clients in the client role. Clients go on every host, installed via pip. This setting here would install python-ironicclient via apt, resulting in confusion over which version is actually installed and used.